xml/System/Enum.xml

```diff
@@ -4132,19 +4132,22 @@ thisInstance And flag = flag
 
- The `value` parameter contains the string representation of an enumeration member's underlying value or named constant, or a list of named constants or underlying values delimited by commas (,). If `value` includes multiple named constants or values, one or more blank spaces can precede or follow each value, name, or comma in `value`. If `value` is a list, `result` reflects the value of the specified names or underlying values combined with a bitwise `OR` operation. If `value` is the string representation of the name of an enumeration value, the comparison of `value` with enumeration names is case-sensitive.
+<xref:System.Enum.TryParse%60%601%28System.String%2C%60%600%40%29> is identical to the <xref:System.Enum.Parse%28System.Type%2CSystem.String%29> method, except that instead of throwing an exception, it returns `false` if the conversion fails. It eliminates the need for exception handling when parsing the string representation of an enumeration value.
 
- If `value` is a name that does not correspond to a named constant of `TEnum`, the method returns `false`. If `value` is the string representation of an integer that does not represent an underlying value of the `TEnum` enumeration, the method returns an enumeration member whose underlying value is `value` converted to an integral type. If this behavior is undesirable, call the <xref:System.Enum.IsDefined%2A> method to ensure that a particular string representation of an integer is actually a member of `TEnum`.
+The `value` parameter contains the string representation of an enumeration member's underlying value or named constant, or a list of named constants or underlying values delimited by commas (,). If `value` includes multiple named constants or values, one or more blank spaces can precede or follow each value, name, or comma in `value`. If `value` is a list, `result` reflects the value of the specified names or underlying values combined with a bitwise `OR` operation. If `value` is the string representation of the name of an enumeration value, the comparison of `value` with enumeration names is case-sensitive.
+
+If `value` is a name that does not correspond to a named constant of `TEnum`, the method returns `false`. If `value` is the string representation of an integer that does not represent an underlying value of the `TEnum` enumeration, the method returns an enumeration member whose underlying value is `value` converted to an integral type. If this behavior is undesirable, call the <xref:System.Enum.IsDefined%2A> method to ensure that a particular string representation of an integer is actually a member of `TEnum`.
+
+If the parse operation fails, the `result` parameter contains the default value of the underlying type of `TEnum`. The default value is the member in the enumeration with an assigned value of 0, or the first member of the `TEnum` if no values are assigned, or `0` if values are assigned to each member but none of them is 0.
```

Comment:
This is the new paragraph.

Reply:
Thanks, I think the wording still needs update, because the result will be defaulted to 0; which may or may not match with the underlying value of the Enum.
```suggestion
If the parse operation fails, the `result` parameter contains the default value of 0 which may not be a member of the underlying <c>TEnum</c> enumeration. If no values are assigned to named constant of `TEnum` default is equal to the first member of the `TEnum` else it is equal to the member in the enumeration with an assigned value of 0.
```